### PR TITLE
Added OATH and GRAIN tokens

### PR DESCRIPTION
**Pull request by 0xTollywood**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Review by 0xToli — COMMENTED** [review]
switched to grain-granary to match coinpaprika

**Comment by 0xToli** [comment]
[oath on bnb ](https://bscscan.com/address/0xd3c6ceedd1cc7bd4304f72b011d53441d631e662) 
[oath on avax](https://snowtrace.io/token/0x2c69095d81305f1e3c6ed372336d407231624cea)
[grain on eth](https://etherscan.io/address/0xF88Baf18FAB7e330fa0C4F83949E23F52FECECce)

**Review by 0xToli — COMMENTED** [review]
checked id and correwct addresses

OATH:

Arbitrum: https://arbiscan.io/address/0xa1150db5105987cec5fd092273d1e3cbb22b378b
BSC: https://bscscan.com/address/0xd3c6ceedd1cc7bd4304f72b011d53441d631e662
KAVA: https://explorer.kava.io/address/0x7D3B448c1242b563c65e8DAfCc13557c3069B5b6
Polygon: https://polygonscan.com/address/0xc2c52ff5134596f5ff1b1204d3304228f2432836
Optimism: https://optimistic.etherscan.io/address/0x39fde572a18448f8139b7788099f0a0740f51205
Ethereum: https://etherscan.io/address/0x6f9c26fa731c7ea4139fa669962cf8f1ce6c8b0b
Fantom: https://ftmscan.com/token/0x21ada0d2ac28c3a5fa3cd2ee30882da8812279b6
Avalanche: https://snowtrace.io/token/0x2c69095d81305f1e3c6ed372336d407231624cea 

GRAIN:
      "Fantom": "0x02838746d9E1413E07EE064fcBada57055417f21",
      "Optimism": "0xfD389Dc9533717239856190F42475d3f263a270d",
      "Metis": "0xE1537feF008944D1c8DcAfBAcE4DC76D31D22DC5",
      "Arbitrum": "0x80bB30D62a16e1F2084dEAE84dc293531c3AC3A1",
      "Avalanche": "0x9df4Ac62F9E435DbCD85E06c990a7f0ea32739a9",
      "Ethereum": "0xF88Baf18FAB7e330fa0C4F83949E23F52FECECce",
      "Binance": "0x8F87A7d376821c7B2658a005aAf190Ec778BF37A",
      "Polygon": "0x8429d0AFade80498EAdb9919E41437A14d45A00B"

File: models/prices/avalanche_c/prices_avalanche_c_tokens.sql [hunk 1]
```diff
@@ -48,6 +48,8 @@ FROM
     ("mimatic-mimatic", "avalanche_c", "MAI", "0x5c49b268c9841aff1cc3b0a418ff5c3442ee3f3b", 18),
     ("luna-luna-wormhole", "avalanche_c", "LUNA", "0x70928e5b188def72817b7775f0bf6325968e563b", 6),
     ("ust-terrausd-wormhole", "avalanche_c", "UST", "0xb599c3590f42f8f995ecfa0f85d2980b76862fc1", 6),
-    ("ico-axelar", "avalanche_c", "AXL", "0x44c784266cf024a60e8acf2427b9857ace194c5d", 6)
+    ("ico-axelar", "avalanche_c", "AXL", "0x44c784266cf024a60e8acf2427b9857ace194c5d", 6),
+    ("grain-granary","avalanche_c","GRAIN","0x9df4ac62f9e435dbcd85e06c990a7f0ea32739a9",18),
+    ("oath-oath","avalanche_c","OATH","0x2c69095d81305f1e3c6ed372336d407231624cea",18)
```
Comment (Hosuke) [inline models/prices/avalanche_c/prices_avalanche_c_tokens.sql:53]: https://snowtrace.io/token/0x2c69095d81305f1e3c6ed372336d407231624cea

File: models/prices/bnb/prices_bnb_tokens.sql [hunk 2]
```diff
@@ -197,7 +197,9 @@ FROM
     ("stg-stargatetoken","bnb","STG","0xb0d502e938ed5f4df2e681fe6e419ff29631d62b",18),
     ("ico-axelar", "bnb", "AXL", "0x8b1f4432f943c465a973fedc6d7aa50fc96f1f65", 6),
     ("lvl-level","bnb","LVL","0xb64e280e9d1b5dbec4accedb2257a87b400db149",18),
-    ("ghny-grizzly-honey","bnb","GHNY","0xa045e37a0d1dd3a45fefb8803d22457abc0a728a",18)
+    ("ghny-grizzly-honey","bnb","GHNY","0xa045e37a0d1dd3a45fefb8803d22457abc0a728a",18),
+    ("grain-granary","bnb","GRAIN","0x8f87a7d376821c7b2658a005aaf190ec778bf37a",18),
+    ("oath-oath","bnb","OATH","0xd3c6ceedd1cc7bd4304f72b011d53441d631e662",18)
```
Comment (Hosuke) [inline models/prices/bnb/prices_bnb_tokens.sql:202]: https://bscscan.com/token/0xd3c6ceedd1cc7bd4304f72b011d53441d631e662

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 3]
```diff
@@ -1638,7 +1638,9 @@ FROM
     ('imgnai-imgnai', 'ethereum', 'IMGNAI', '0xA735A3AF76CC30791C61c10d585833829d36CBe0', 9),
     ('woxen-wrapped-oxen', 'ethereum', 'WOXEN', '0xd1e2d5085b39B80C9948AeB1b9aA83AF6756bcc5', 9),
     ('xrio-realio-network-utility-token-exchangable', 'ethereum', 'XRIO', '0x0B4AC2BA3D4924C9A06D9C1d08D7867059A39cC1', 18),
-    ('pepe-pepe1', 'ethereum', 'PEPE', '0x6982508145454Ce325dDbE47a25d4ec3d2311933', 18)      
+    ('pepe-pepe1', 'ethereum', 'PEPE', '0x6982508145454Ce325dDbE47a25d4ec3d2311933', 18),   
+    ('grain-granary','ethereum','GRAIN','0xf88baf18fab7e330fa0c4f83949e23f52fececce',18)
```
Comment (Hosuke) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1642]: https://etherscan.io/token/0xF88Baf18FAB7e330fa0C4F83949E23F52FECECce

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @0xTollywood 

API for GRAIN:
https://api.coinpaprika.com/v1/coins/grain-granary

API for OATH:
https://api.coinpaprika.com/v1/coins/oath-oath

**Review by jeff-dude — APPROVED** [review]
thanks both!

**Pull request merged**